libarchive.cabal

```diff
@@ -46,7 +46,6 @@ flag low-memory
 
 flag static
   default:     False
-  manual:      True
```

Comment:
Interesting choice... shouldn't we rather invert it to `system-libarchive`?

Reply:
Fair enough!